### PR TITLE
Manager: fix daily schedules input data check

### PR DESCRIPTION
**Pull request by AenBleidd**
As requested in #2044: Daily schedules not checking hours input

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: clientgui/DlgAdvPreferences.cpp [hunk 1]
```diff
@@ -1011,6 +1011,21 @@ bool CDlgAdvPreferences::IsValidTimeValue(const wxString& value) {
             return false;
         }
     }
+    //verify correct diapasone
```
Comment (ChristianBeer) [inline clientgui/DlgAdvPreferences.cpp]: This is hard to translate for non English speakers. I had to look it up in the Oxford dictionary and found out it means "range, or scope". A more simpler sentence would be better here.

"verify correct format and range of time values"

Comment (AenBleidd) [reply clientgui/DlgAdvPreferences.cpp]: Fixed

**Comment by AenBleidd** [comment]
@davidpanderson, could you please review and merge this? Thanks

**Comment by AenBleidd** [comment]
@davidpanderson, could you please take a look at this pull request too? Thanks

**Comment by davidpanderson** [comment]
Thanks!

**Pull request merged**